src/app/shared/shared.module.ts

```diff
@@ -395,6 +398,8 @@ const COMPONENTS = [
   ItemSubmitterComponent,
   ItemDetailPreviewComponent,
   ItemDetailPreviewFieldComponent,
+  ItemBackButtonComponent,
+  ThemedItemBackButtonComponent,
```

Comment:
Rather than adding these to the "shared" module, could we add them to the `item-page.module.ts` instead?  It appears they are specific to the ItemPageModule (which is also used by all Entity pages as well), so they don't need to be in the "SharedModule"

Reply:
Sure, I can look into this more. 

We may be able to use the `item-shared-module.ts`. That module is imported by the `entity-group` modules. Also, I just added another wrinkle here. I decided that because the button is now a themed component it makes sense use it for the `browse-by` component back buttons.  If @artlowel agrees with this approach then we would need to add the `ItemSharedModule` to `browse-by.module.ts` as well.

Reply:
The solution I came up with was more complex than I guessed. Because the button is also used for `browse-by` components I decided to make separate modules for both the results back button and the shared browse-by component. That removes both from the "shared" module. 

I know that the Atmire team is working on shrinking the size of `shared.module` and the approach taken here might not align well with what they are thinking. If so, that's fine. I can fix things here if needed.

The back buttons are working now for Admin Search and MyDSpace.